hiddenEdit.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+session_start();
+require_once 'functions.php';
+
+$id = targetID();
+$db = getDb();
+$editSeason = filter_var($_POST['editSeason'], FILTER_VALIDATE_INT);
```

Comment:
if this isnt a valid INT this function will return false. your not doing any check on if that happens

Reply:
'if' statement written to request an integer from user